stdlib/LinearAlgebra/src/uniformscaling.jl

```diff
@@ -227,6 +227,14 @@ Broadcast.broadcasted(::typeof(*), J::UniformScaling,x::Number) = UniformScaling
 
 Broadcast.broadcasted(::typeof(/), J::UniformScaling,x::Number) = UniformScaling(J.λ/x)
 
+(^)(A::UniformScaling, x::Number) = UniformScaling((A.λ)^x)
```

Comment:
Just to keep other people busy: in the lines above and below, we seem to be using `J` for `UniformScaling`s consistently. Maybe use `J` here as well to not confuse the eye?

Reply:
Done.